src/Collections/TerminusCollection.php

```diff
@@ -91,6 +94,22 @@ public function all()
     public function fetch()
     {
         foreach ($this->getData() as $id => $model_data) {
+            if (!is_object($model_data)) {
+                // For some reason I can't replicate, occasionally $model_data is just a string here.
+                $trace = debug_backtrace();
+                $error_message = "Fetch failed {file}:{line} model_data expected as object but returned as {type}.";
+                $context = [
+                    'file' => $trace[0]['file'],
+                    'line' => $trace[0]['line'],
+                    'type' => gettype($model_data),
+                ];
+                if (is_string($model_data)) {
```

Comment:
let's try this again...

```suggestion
                if (is_scalar($model_data)) {
```
@greg-1-anderson how do you feel about is_scalar() here? I tested locally with php -r and it does implicit string conversion for integer, float, and boolean values in the string functions on line 108. float and integer behave predictably. boolean becomes "1" for true and "" for false.

Reply:
Sure, but if we do that, then maybe we should use `var_export($model_data)` to output the value?
```
php > print var_export(true);
true
php > print var_export(false);
false
php > print var_export(0);
0
php > print var_export("0");
'0'
```
Maybe do length checking with a nested `if`:
```
if (is_string($model_data) && strlen($model_data) > 250) {
    $model_data = substr($model_data, 0, 250);
}
```

Reply:
I mentioned in slack that I'd consider doing `print_r($model_data, true)` if `$model_data` wasn't scalar.  If it's scalar, `var_export` or similar isn't necessary because it can simply be treated as a string.  If you're going to `var_export` integers why not just `var_export` everything ;)

Is `var_export` superior to `print_r`?   I see people use `var_dump` a lot and I find `print_r` a lot nicer.